### PR TITLE
Fix misspelling

### PR DESCRIPTION
**Pull request by nest1ing**
This brings fixes for all misspells found through all sources.

Signed-off-by: Alexander Filippov <a.filippov@yadro.com>

### CODE REVIEW

**Review by artemsen — DISMISSED** [review]

File: src/image_intel.cpp [hunk 1]
```diff
@@ -83,9 +83,9 @@ static MountPoints getMountPoints()
 
 static void unmountFilesystem(const std::string& filesystem)
 {
-    // Some systemd services may to occupate the RW partition
+    // Some systemd services may occupy the RW partition
     // and make delay up to 20 seconds. We should to wait them
```
Comment (AlexanderAmelkin) [inline src/image_intel.cpp]: `resulting in a delay of up to 20 seconds. We should wait for them`

File: src/image_intel.cpp [hunk 2]
```diff
@@ -83,9 +83,9 @@ static MountPoints getMountPoints()
...
-    // befor throw an error.
+    // before throw an error.
```
Comment (AlexanderAmelkin) [inline src/image_intel.cpp]: `before throwing an error`

**Review by AlexanderAmelkin — CHANGES_REQUESTED** [review]

File: src/image_intel.cpp (hunk 1 above)
Comment (nest1ing) [reply src/image_intel.cpp]: done

File: src/image_intel.cpp (hunk 2 above)
Comment (nest1ing) [reply src/image_intel.cpp]: done

**Review by AlexanderAmelkin — APPROVED** [review]

**Pull request merged**